### DISCUSSION TITLE
In what form does the VST3 SDK need to be provided to Audacity during the build process?

### DISCUSSION THREAD

**Discussion by schuelermine** [audacity/audacity#3584] [Development]
Hi, I’m trying to build the recent 3.2.0-alpha-1 version from source and I’m getting this error:

```
CMake Error at cmake-proxies/cmake-modules/AudacityDependencies.cmake:238 (message):
  Failed to find the system package vst3sdk
Call Stack (most recent call first):
  cmake-proxies/CMakeLists.txt:26 (add_conan_lib)
```

In what form does this need to be provided? I believe that I somehow need to use https://github.com/steinbergmedia/vst3sdk, but it doesn’t look like you can build that tree from source—the instructions only specify how to build the examples from source.

**Comment by schuelermine** [comment]
I found further instructions here: https://steinbergmedia.github.io/vst3_dev_portal/pages/Getting+Started/How+to+setup+my+system.html#for-linux

**Reply by crsib** [reply]
It is not currently possible to build VST3 support without Conan

**Reply by schuelermine** [reply]
Oh OK

**Reply by schuelermine** [reply]
Thank you
